### ISSUE TITLE
Reduce of empty array with no initial value error in System Overview page

### ISSUE THREAD

**Issue by edmundoa** [Graylog2/graylog2-server#7329]
I had today same error when visiting `https://Roman/system/overview` with different Stack Trace 
```
reduce of empty array with no initial value

Stack Trace:

render@https://Roman/assets/app.9019bbb55e618ace00d4.js:68:541053
go@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:65535
yo@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:65330
Qs@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:99620
Is@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:83908
Rs@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:80933
Ps@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:79506
_i/<@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:41771
29/exports.unstable_runWithPriority@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:42:3998
fi@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:41500
_i@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:41719
mi@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:41652
Ds@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:78497
enqueueSetState@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:18:46463
24/b.prototype.setState@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:26:1438
n@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:10:1594
614/a.prototype.emit@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:78:243907
trigger@https://Roman/assets/vendor.5452e7a2fdfb6282da0a.js:10:2469
list/r<@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:68:8092
l@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:90514
YPyIcTeH/</</<[22]</r.exports/O.prototype._settlePromiseFromHandler@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:63769
YPyIcTeH/</</<[22]</r.exports/O.prototype._settlePromise@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:64569
YPyIcTeH/</</<[22]</r.exports/O.prototype._settlePromise0@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:65268
YPyIcTeH/</</<[22]</r.exports/O.prototype._settlePromises@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:66595
YPyIcTeH/</</<[2]</s.prototype._drainQueue@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:20459
YPyIcTeH/</</<[2]</s.prototype._drainQueues@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:20520
s/this.drainQueues@https://Roman/assets/builtins.9019bbb55e618ace00d4.js:76:18743


Component Stack:

    in SystemJobsComponent
    in h
    in ConnectStoresWrapper[h] stores=currentUser
    in span
    in t
    in t
    in n
    in div
    in t
    in div
    in t
    in div
    in l
    in div
    in i
    in t
    in withRouter(t)
    in c
    in d
    in ConnectStoresWrapper[d] stores=currentUser
    in t
    in withRouter(t)
    in RouterContext
    in Router
    in m
    in t
    in n
    in t
    in c
    in AppFacade
```

    Graylog Version: 3.2.0-3.beta.3
    Elasticsearch Version: 6.8.6-1
    MongoDB Version: 4.0.14-1.el7
    Operating System: CentOS 7
    Browser version: Firefox 72.0.2

_Originally posted by @qarmin in https://github.com/Graylog2/graylog2-server/issues/7056#issuecomment-580211193_

**Comment by qarmin** [comment]
In closed issue I wasn't very clear about log.
This log which I attached([Log](https://github.com/Graylog2/graylog2-server/files/4133820/FFFFFFFFFFFFFFFFFFF.txt)), have info about things which happens when Graylog crashes.
I also attached logs from yesterday, because this logs contains errors with Mongodb and I think that is probably that Mongodb errors could help to crash Graylog. 

List of steps which I done before error appear:
1. I installed Sidecar on CentOS machine
2. Later I set server name, token api and disabled tls checking in settings yml file
3. A little after that(5 min - 15 min) Graylog starts to lags/loads infinitely pages
4. After a while a page like below was showed 
5. I collected the logs


![2](https://user-images.githubusercontent.com/4478521/71349988-908dd300-2570-11ea-9461-0bc7bc268b8c.png)

**Comment by edmundoa** [comment]
Your first comment (the one I used to open this issue) describes a defect in the UI of our product that we need to address and I opened this ticket to track it. For what I could see so far, that situation would only happen when a Graylog server is alive but very slow.

The other information you are sharing with us may have caused this issue to appear in your system, it may even be a bug in Graylog, but it is not the same issue as the one described above. As I mentioned in the other ticket, if you are unsure what happened and need some help debugging the problem the [Graylog discussion forum](https://community.graylog.org/) is the right place to post that.

Thank you!
